examples/browser-to-server/package.json

```diff
@@ -5,14 +5,20 @@
   "type": "module",
```

Comment:
I think it falls out of scope for this PR, but something to think about is that we might as well turn this repo into a workspace. This way the example and webrtc package could be both living in the same package, allowing the example to make use of the @libp2p/webrtc package the same as any external usage would. Making the example also directly copyable without having to rely on relative paths in its package.json to make it work given that the relative paths would be fixed by the root workspace definition :) Just my 2 cents on this matters, but feels cleaner to me.

Reply:
The pathing has been removed in favor of the npm package import.  The example was written before the package was released, so this was something that needed to be done now that it's published.  Thanks for the reminder!  

Regarding the placement of this comment, is there a concern with `"type": "module"`?

Reply:
No no, just decided to put it on top of the file it was related to. Sorry for the confusion there.